### PR TITLE
[svg4everybody] introduce typings

### PR DESCRIPTION
**Pull request by BendingBender**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not provide its own types, and you can not add them.
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, and `strictNullChecks` set to `true`.

### CODE REVIEW

File: types/svg4everybody/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
+// Type definitions for svg4everybody 2.1
+// Project: https://github.com/jonathantneal/svg4everybody#readme
+// Definitions by: BendingBender <https://github.com/BendingBender>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import { Svg4everybodyOpts } from "svg4everybody";
```
Comment (weswigham) [inline types/svg4everybody/index.d.ts]: This import's very confusing. Most packages shouldn't import themselves. Remove it.

File: types/svg4everybody/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+export as namespace svg4everybody;
+
+export = svg4everybody;
+
+declare function svg4everybody(options?: Svg4everybodyOpts): void;
```
Comment (weswigham) [inline types/svg4everybody/index.d.ts]: Then replace `Svg4everybodyOpts` on this line with `svg4everybody.Svg4everybodyOpts`.

**Review by weswigham — CHANGES_REQUESTED** [review]
Small comment on a slightly confusing part of your definition that it'd be nice to have fixed.

**Comment by BendingBender** [comment]
Done, please review again.

**Review by weswigham — APPROVED** [review]

**Pull request merged**